src/components/change-view.jsx

```diff
@@ -93,6 +94,10 @@ export default class ChangeView extends React.Component {
   handleToVersionChange (version) {
     this._changeSelectedVersions(null, version);
   }
+  handleLatestToBaseChange () {
```

Comment:
Blank line here, please!

Reply:
Could you elaborate? I really don't understand where to place the blank line and why so I'm not sure what's being asked for here ??

Reply:
Oh, sorry, I thought it was clear—between the two methods (`handleToVesionChange` and `handleLatestToBaseChange`). We always keep a space between function/method definitions.

Reply:
Ah, it's the code inconsistency causing confusion for me again. The previous definitions didn't have spaces, so I added it for them as well.

Reply:
Oh, sorry, could not see in the diff that this was already a problem with the preceding two function. 👍